examples/inverse_kinematics/inverse_kinematics/send_inverse_kinematics_requests.py

```diff
@@ -0,0 +1,361 @@
+            )
+
+            # Attempt to move to each of the desired tool pose to check the IK results.
+            stand_command = None
+            if ik_response.status == inverse_kinematics_pb2.InverseKinematicsResponse.Status.STATUS_OK:
+                self._logger.info("Solution found")
+                self._publish_transform(task_frame_name, "T" + str(i) + "-YES", task_T_desired_tool)
+
+                odom_T_desired_body = get_a_tform_b(
+                    ik_response.robot_configuration.transforms_snapshot,
+                    ODOM_FRAME_NAME,
+                    BODY_FRAME_NAME,
+                )
+                mobility_params = robot_command_pb2.MobilityParams(
+                    body_control=robot_command_pb2.BodyControlParams(
+                        body_pose=RobotCommandBuilder.body_pose(ODOM_FRAME_NAME, odom_T_desired_body.to_proto())
+                    )
+                )
+                stand_command = RobotCommandBuilder.synchro_stand_command(params=mobility_params)
+            elif ik_response.status == inverse_kinematics_pb2.InverseKinematicsResponse.Status.STATUS_NO_SOLUTION_FOUND:
+                self._logger.info("No solution found")
+                self._publish_transform(task_frame_name, "T" + str(i) + "-NO", task_T_desired_tool)
+                stand_command = body_assist_enabled_stand_command
+            else:
+                self._logger.info("Status unknown")
+                self._publish_transform(task_frame_name, "T" + str(i) + "-NO", task_T_desired_tool)
+                stand_command = body_assist_enabled_stand_command
+
+            # Move the arm tool to the requested position.
+            arm_command = RobotCommandBuilder.arm_pose_command_from_pose(
+                hand_pose=(odom_T_task * task_T_desired_tool).to_proto(),
+                frame_name=ODOM_FRAME_NAME,
+                seconds=1,
+                build_on_command=stand_command,
+            )
+            arm_command.synchronized_command.arm_command.arm_cartesian_command.wrist_tform_tool.CopyFrom(
+                wr1_T_tool.to_proto()
+            )
+            arm_command_goal = RobotCommand.Goal()
+            conv.convert_proto_to_bosdyn_msgs_robot_command(arm_command, arm_command_goal.command)
+            result = self._robot_command_client.send_goal_and_wait(
+                action_name="arm_move_one", goal=arm_command_goal, timeout_sec=5
+            )
+
+        # Dock robot.
+        self._logger.info("Docking the robot")
+        result = self._dock(self._dock_id)
+        if not result:
+            self._logger.error("Unable to dock the robot")
+            return False
+
+        # Power off robot.
+        self._logger.info("Powering robot off")
+        result = self._robot.command("power_off")
+        if not result:
+            self._logger.error("Unable to power off robot")
+            return False
```

Comment:
I did all this for safety reasons and for easy of testing.

To move the robot tool to a given position, the robot must have some free space around, and not be on a dock. Because I move the robot in front of the dock I also though it was a good idea to move it back.
We can remove all the code to undock, move, rotate and dock back the robot, but I think we should add some warning on the documentation then.

Reply:
All additional robot moves have been removed.